modules/photo/src/seamless_cloning.cpp

```diff
@@ -79,6 +79,12 @@ void cv::seamlessClone(InputArray _src, InputArray _dst, InputArray _mask, Point
 
     Rect roi_s = boundingRect(mask);
     if (roi_s.empty()) return;
+
+    if (p.x < 0)
+        p.x = dest.cols / 2;
+    if (p.y < 0)
+        p.y = dest.rows / 2;
+
     Rect roi_d(p.x - roi_s.width / 2, p.y - roi_s.height / 2, roi_s.width, roi_s.height);
```

Comment:
Need to ensure that Rect args are still valid.

Reply:
How can I pull sturkmen72's code in order to work on this?